src/jwplayer/utils/iasTracker.ts

```diff
@@ -0,0 +1,25 @@
+import { scriptLoader } from 'jwplayer/utils/scriptLoader';
+
+const scriptUrl = '//static.adsafeprotected.com/vans-adapter-google-ima.js';
+class IasTracker {
+	private scriptPromise: Promise<Event>;
+
+	public async loadIasTrackerIfEnabled(skin: 'fandomdesktop' | 'fandommobile'): Promise<void> {
+		const deserializedConfigData = JSON.parse(localStorage.getItem(`instant-config-${skin}`)).data;
```

Comment:
`Storage.getItem()` returns a key's value or `null` if the key doesn't exist. Just a heads up, `JSON.parse` expects `string`, so it might be a good idea to add a check or handle this case separately.

Reply:
sure